tests/apps/search/test_indexers_courses.py

```diff
@@ -193,3 +202,331 @@ def test_format_es_course_for_api(self):
                 "title": "Duis eu arcu erat",
             },
         )
+
+    def test_build_es_query_search_all_courses(self):
+        # Build a request stub
```

Comment:
All your methods in this file have no docstring. If there is no error it means that the linting is not done on these files!?